lib/fog/ovirt/requests/compute/activate_volume.rb

```diff
@@ -7,6 +7,9 @@ def activate_volume(id, options)
           raise ArgumentError, "volume id is a required parameter for activate-volume" unless options.key? :id
 
           client.activate_volume(id, options[:id])
+
+        rescue ::OVIRT::OvirtException => e
+          raise ::Fog::Ovirt::OvertError, e.message
```

Comment:
Why are we sending only the message? I would like to see `::Fog::Ovirt::OvertError` wrapping the complete `OvirtException` instance, so I will be able to see the real source of the error.

Reply:
done.